doc/source/internals/ondisk/sequences.rst

```diff
@@ -0,0 +1,402 @@
+.. _sequence-internals:
+
+Sequence types
+==============
+
+Both lists and tuples share the same binary representation and are thus binary-compatible.
+
+.. code-block:: C
+
+    union VarLength {
+        // For typecodes l, L, d, t, T
+        long longlength:56;
+
+        // For other typecodes, length < 0xFFFFFF
+        int shortlength:24;
+
+        struct {
+            int padding:24; // 0xFFFFFF
```

Comment:
What purpose does this `padding` member serve ?

Reply:
It helps identify long lengths when reading the buffer. Think of this as rice coding.